www/front_src/src/components/navigation/index.js

```diff
@@ -197,69 +174,65 @@ class NavigationComponent extends Component {
             </span>
           </div>
           <ul
-            class="menu menu-items list-unstyled components"
-            onMouseLeave={this.mouseLeftTheMenu}
+            className={`menu menu-items list-unstyled components`}
           >
             {Object.entries(menuItems).map(([levelOneKey, levelOneProps]) => (
               levelOneProps.label ? (
                 <li
-                  onMouseOver={this.mouseIsMovingOverTheMenu.bind(this, levelOneProps.label)}
-                  class={"menu-item" + (levelOneProps.active ? " active" : "")}
+                  className={`menu-item ${(levelOneProps.toggled && active || levelOneProps.active) ? activated : " to-hover"}`}
                 >
                 <span
-                  onDoubleClick={() => {this.handleDoubleClick(levelOneKey, levelOneProps)}}
-                  onClick={() => {this.collapseLevelTwo(levelOneKey)}}
-                  style={{ cursor: "pointer" }}
-                  class="menu-item-link dropdown-toggle"
+                  onDoubleClick={() => {this.handleDirectClick(levelOneKey, levelOneProps)}}
+                  className={`menu-item-link dropdown-toggle`}
                   id={"menu" + levelOneKey}
                 >
                   <span class={`iconmoon icon-${levelOneProps.menu_id.toLowerCase()}`}>
-                    <span class={"menu-item-name"}>
+                    <span className={`menu-item-name`}>
                       <Translate value={levelOneProps.label}/>
                     </span>
                   </span>
                 </span>
                 <ul
-                  class="collapse collapsed-items list-unstyled"
+                  className={`collapse collapsed-items list-unstyled ${(levelOneProps.toggled && active) ? activated : " " }`}
                   style={{ display: (levelOneProps.toggled && active) ? "block" : "none" }}
                 >
                   {Object.entries(levelOneProps.children).map(([levelTwoKey, levelTwoProps]) => {
                     const levelTwoUrl = this.getUrlFromEntry(levelTwoKey, levelTwoProps);
                     if (levelTwoProps.label) {
                       return (
                         <li
-                          class={
-                            "collapsed-item" +
-                            (levelTwoProps.collapsed || (this.isActive(pageId, levelTwoUrl))
-                              ? " active"
-                              : ""
-                            )
+                          className={
+                            `collapsed-item
+                            ${levelTwoProps.collapsed || (this.isActive(pageId, levelTwoUrl))
+                            ?
```

Comment:
bad coding style. Should be : 
expression
  ? result1
  : result2

Reply:
done